azure/README.md

```diff
@@ -0,0 +1,73 @@
+<!--
+Copyright (c) 2012 - 2016 YCSB contributors. All rights reserved.
```

Comment:
I've updated my code according to your review.
I'm not sure why the new commit version can't pass JDK:openjdk7. I'm checking.

For the difference between azure documentDB and azure table storage, please kindly refer to 
http://stackoverflow.com/questions/28928016/microsoft-azure-documentdb-vs-azure-table-storage